test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -897,20 +1037,20 @@ def test_cgroup_cpuset_and_memory(self):
         ehost = j.attributes['exec_host']
         tmp_file = filename.split(':')[1]
         tmp_host = ehost.split('/')[0]
-        tmp_out = self.wait_and_read_file(filepath=tmp_file, mom=tmp_host)
-        self.logger.info("%s: %s" % (name, tmp_out))
+        tmp_out = self.wait_and_read_file(filename=tmp_file, mom=tmp_host)
+        self.logger.info('%s: %s' % (name, tmp_out))
```

Comment:
Is this a debugging message, or a log message that is useful for the test.  I find useful log messages are saying what the test is doing, not values from variables.  Those are useful to debug a test, but I try and remove that type of log message when the test is complete.  What you want to print is up to you though.

Reply:
These messages were for test debugging. I have removed them.